client/web/compose/src/components/PageBlocks/RecordListBase.vue

```diff
@@ -1071,12 +1078,22 @@ export default {
     onFilter (filter = []) {
       filter.forEach(f => {
         if (this.activeFilters.includes(f.name)) {
-          this.filterPresets.find(p => p.name === f.name).filter.forEach((filterPreset) => {
-            if (!isEqual(f.filter, filterPreset.filter)) {
-              const filterIndex = this.activeFilters.indexOf(f.name)
-              this.activeFilters.splice(filterIndex, 1)
-            }
-          })
+          const filterPresets = this.filterPresets.find(p => p.name === f.name)
+
+          if (filterPresets) {
+            filterPresets.filter.forEach((filterPreset) => {
+              if (!isEqual(f.filter, filterPreset.filter)) {
+                const filterIndex = this.activeFilters.indexOf(f.name)
+                this.activeFilters.splice(filterIndex, 1)
+
+                this.activeFilters.push(this.$t('recordList.customFilter'))
```

Comment:
Since the same code is in lines 1095 and 1096 it makes me think you could rewrite this to only use the code once (lines 1089/1090 are same as 1095/1096)